.github/workflows/openfire-plugin-build.yml

```
@@ -51,7 +51,9 @@ jobs:
         id: set-java-targets
         run: |
           set -e
-          if [ "${{steps.get-java-version-from-plugin-xml.outputs.info}}" == "11" ] || [[ "${{steps.get-openfire-version-from-plugin-xml.outputs.info}}" =~ ^4\.8.*$ ]]; then
+          if [ "${{steps.get-java-version-from-plugin-xml.outputs.info}}" == "17" ] || [[ "${{steps.get-openfire-version-from-plugin-xml.outputs.info}}" =~ ^4\.10\..*$ ]] || [[ "${{steps.get-openfire-version-from-plugin-xml.outputs.info}}" =~ ^5\..*\..*$ ]]; then
```

```suggestion
          if [ "${{steps.get-java-version-from-plugin-xml.outputs.info}}" == "17" ] || [[ "${{steps.get-openfire-version-from-plugin-xml.outputs.info}}" =~ ^((4\.10\..*)|(5\..*\..*)$ ]]; then
```
Is shorter more complicated regex better?

---

I don't know. I don't have a strong preference either way.